CONTRIBUTING.md

```diff
@@ -6,8 +6,7 @@ We appreciate your help!
 
 ## Before filing an issue
 
-If you are unsure whether you have found a bug, please consider asking in the [vald slack](https://vald-community.slack.com/messages/CN1RDC8NB) first. If
-the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the issue tracker.
+If you are unsure whether you have found a bug, please consider asking in the [vald slack](https://vald-community.slack.com/messages/CN1RDC8NB) first. If the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the issue tracker.
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
The link to Vald Slack should be the invitation link.
```suggestion
If you are unsure whether you have found a bug, please consider asking in the [Vald Slack](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) first. If the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the issue tracker.
```

Reply:
thx. I'll fix